flink-runtime/src/test/java/org/apache/flink/runtime/leaderretrieval/SettableLeaderRetrievalServiceTest.java

```diff
@@ -32,6 +32,7 @@
 public class SettableLeaderRetrievalServiceTest extends TestLogger {
 
     private SettableLeaderRetrievalService settableLeaderRetrievalService;
+    private static final long TIMEOUT = 50L;
```

Comment:
```suggestion
    private static final long TIMEOUT_IN_MILLIS = 50L;
```
It's always a good habit to add the unit to variables like that.